pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.novartis.opensource</groupId>
+  <groupId>com.novartis.opensource.aspirin</groupId>
```

Comment:
could we avoid this subgrouping ?

Reply:
That's weird this groups should've been removed already, let me double check